scripts/typecheck-all.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-for name in $(find packages -name node_modules -prune -o -name 'cardpay-subgraph' -prune -o -name 'tsconfig.json' -print); do
+for name in $(find packages -name node_modules -prune -o -name '*-subgraph' -prune -o -name 'tsconfig.json' -print); do
```

Comment:
Was getting a lot of typecheck errors (can bee seen here - https://github.com/cardstack/cardstack/actions/runs/3436424201/jobs/5729924858)  so I followed this pattern where we skip the checking for subgraph related packages.